### PR TITLE
[CI] Quantization workflow fix

### PR DESCRIPTION
**Pull request by SunMarc**
# What does this PR do ? 

This PR make sure that the setup happens before the quantization tests.  cc @younesbelkada 

The tests are triggered this time: 
Screenshot 2024-04-10 at 11 22 25 AM

### CODE REVIEW

File: .github/workflows/self-scheduled-caller.yml [hunk 1]
```diff
@@ -7,7 +7,8 @@ on:
     - cron: "17 2 * * *"
   push:
     branches:
-      - run_scheduled_ci*
+      # - run_scheduled_ci*
+      - fix-ci-quantization
```
Comment (younesbelkada) [inline .github/workflows/self-scheduled-caller.yml:11]:
```suggestion
      - run_scheduled_ci*
```

**Review by younesbelkada — APPROVED** [review]
Thanks !

File: .github/workflows/self-scheduled-caller.yml (hunk 1 above)
Comment (SunMarc) [reply .github/workflows/self-scheduled-caller.yml:11]: yes ! just testing to see if everything works as expected this time ;)

**Comment by SunMarc** [comment]
merging this since this is a tiny change

**Pull request merged**